config/settings/sandbox.yml

```diff
@@ -0,0 +1,16 @@
+# The url for this app, also used by `dfe_sign_in`
+base_url: https://www.register-trainee-teachers.education.gov.uk
```

Comment:
Isn't it `https://www.register-trainee-teachers.education.gov.uk` ?

Reply:
done..